runtime/src/bank.rs

```diff
@@ -7651,25 +7635,17 @@ impl Bank {
         new_feature_activations: &HashSet<Pubkey>,
     ) {
         let feature_set = self.feature_set.clone();
-        let should_apply_action_for_feature_transition = |feature_id: &Pubkey| -> bool {
-            if only_apply_transitions_for_new_features {
-                new_feature_activations.contains(feature_id)
-            } else {
-                feature_set.is_active(feature_id)
-            }
-        };
 
         let builtin_feature_transitions = self.builtin_feature_transitions.clone();
         for transition in builtin_feature_transitions.iter() {
-            if let Some(builtin_action) =
-                transition.to_action(&should_apply_action_for_feature_transition)
-            {
-                match builtin_action {
-                    BuiltinAction::Add(program_id, builtin) => {
-                        self.add_builtin(program_id, builtin)
-                    }
-                    BuiltinAction::Remove(program_id) => self.remove_builtin(&program_id),
```

Comment:
Do we not support removals of builtin?

Reply:
See PR description: "instead one should now overwrite a built-in program with a tombstone." but I can also add that as a comment in code, question is where would it go?

Reply:
Ok, makes sense. Adding tombstone will be handled in a separate PR?

Reply:
Brought back the method and implemented it using tombstones.